wp-content/themes/wpdemo/functions.php

```diff
@@ -118,9 +118,137 @@ function send_email_to_admin( $user_id, $old_data, $new_data ) {
 }
 
 
+function dx_location_box() {
+    $screens = [ 'students' ];
+    foreach ( $screens as $screen ) {
+        add_meta_box(
+            'dxloc_box_id',                 
+            'Location',     
+            'dx_lo_box_html',  
+            $screen                    
+        );
+    }
+}
```

Comment:
You don't need to put add_meta_box inside an arrays, as the $screen arguments accepts arrays.
Although, it's nice you thought about that.

https://developer.wordpress.org/reference/functions/add_meta_box/